target_chains/ethereum/sdk/solidity/PythUtils.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+library PythUtils {
+    /// @notice Converts a Pyth price to a uint256 with a target number of decimals
```

Comment:
can you give an example here to clarify what this means (use ethereum and 18 decimals)

Reply:
Can you also note that this function can lose precision and you need to be careful about that? For example, if the price is 0.003 and you set targetdecimals to 2, then the result will be 0.

Reply:
Added the tests [PythTestUtils.t.sol](https://github.com/pyth-network/pyth-crosschain/pull/1390/files#diff-746fac818ac7e04dd794579164481fa11be60cb02c136906a1a216b73e02536f)